mongodb_cmdline_tool/tasks.py

```diff
@@ -161,10 +161,12 @@ def lint(c, eslint=False):
     :param eslint: Run ESLint for JS files. Default: False.
     """
     init(c)
-    with c.cd(str(kHome / 'mongo')):
+    # On first run these tools will install artifacts not in the mongo .gitignore file, so we
+    # run them from within ~/bin to run the binaries from a non-version-controlled location.
+    with c.cd(str(kHome / 'bin')):
```

Comment:
Hm, what error did you encounter with the old version of running from `~/mongo`? My understanding is that cd-ing into bin shouldn't be necessary because we add `~/bin` to PATH on line 138 of `setupenv.py` (the above file)

Reply:
It's been a few weeks, but based on what I wrote in #9  I believe that the problem was really that I ended up with a `clang_format` file that had been temporarily downloaded into the mongo source tree and you recommended installing it into ~/bin instead.  Since the download location is controlled from within the mongo repo, I thought it was better to leave the mongo repo alone and just change this tool.

Reply:
Ah gotcha. That makes sense. I'll create an issue to have clang-format be downloaded to the correct location